### ISSUE TITLE
CLI basic auth username/password flags do not work

### ISSUE THREAD

**Issue by desimaniac** [jiahaog/nativefier#477]
### Description

PR #444 Doesn't work for me. Still get a password prompt for almost every site I try that has basic http authentication.

### Details

- Are you nativefying a *public* website? Using it for personal public facing site's (e.g. NZBGet, Sonarr, etc)
- Full nativefier command used to build your app: `nativefier --basic-auth-username USERNAME --basic-auth-password PASSWORD --name "NZBGet" "https://nzbget.DOMAIN.COM/"  ~/nativefier
`
- Version of Nativefier (run `nativefier --version`): `v7.4.1`
- Version of node.js (run `node --version`): `v6.11.3`
- OS: `macOS 10.11`

**Comment by ronjouch** [comment]
@desimaniac thanks for the feedback. Please share a public reproduction scenario, here this works for me (i.e. the basic http auth provided by the test website passes and I see the auth-requiring image): `nativefier --name "issue477" --basic-auth-username httpwatch --basic-auth-password httpwatch "https://www.httpwatch.com/httpgallery/authentication/authenticatedimage/default.aspx?0.3296613056368104" /tmp/issue477 && /tmp/issue477/issue477-linux-x64/issue477`

**Comment by desimaniac** [comment]
Can I send you an email or hit you up on discord or something ? I'll have you take a look at some sites.

**Comment by desimaniac** [comment]
PS. I created the exactly nativefier you did, and still get the popup ...

![image](https://user-images.githubusercontent.com/5501908/32581292-f1c7d49c-c4ae-11e7-9c71-d773a9694e58.png)


Even tried to change it around: `nativefier --name "issue477" --basic-auth-username httpwatch --basic-auth-passwordnew httpwatch "https://www.httpwatch.com/httpgallery/authentication/authenticatedimage/default.aspx?0.3296613056368104" ~/nativefier`


Same thing. When i login manually, that's when i get ...

![image](https://user-images.githubusercontent.com/5501908/32581341-32685080-c4af-11e7-9567-a1c6b5072d21.png)

**Comment by ronjouch** [comment]
@desimaniac ha! Forgot that these `--basic-auth-{username,password}` flags are super recent and not released yet to npm! So just wait for 7.4.2, or build your nativefier from `master` (it's not that hard/long!), following the [development setup guide](https://github.com/jiahaog/nativefier/blob/master/docs/development.md#environment-setup).

---

@jiahaog hey there! It's been a long time since 7.4.1, could you cut a new release? Here's a readymade summary for the release notes and to update changelog.md 🙂:

```
- Add `--tray` flag to let app running in background on window close. Supports in-title counter. (Issue #304, PR #457)
- Add HTTP `--basic-auth-{username,password}` flags (Issue #275, PR #444)
- Add offline build detection and advice (Issue #448, PR #452)
- Add 'Paste and Match Style' to Edit menu (Issue #404, PR #447)
- Add setting environment variables (PR #419)
- Add `app-copyright`, `app-version`, `build-version`, `version-string` and `win32metadata` flags (Issue #226, PR #244)
- Fix: Make title counter regex match '+' after number, used by certain sites (PR #424)
```

**Comment by desimaniac** [comment]
@ronjouch Thanks! I will do that.
